example/commandline-oauth.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
```

Comment:
Worth adding document right now. Saying that this example is only for certain partners. 

Otherwise this might confuse people. The example/readme part in Github is educational. So we want to make sure they are as clear as possible.

Reply:
Actually will change this to a working example so people can use it today

Reply:
Cool